server/src/controllers/configController.test.ts

```diff
@@ -48,6 +51,9 @@ describe('configController', () => {
     expect(resp.lastJson.waitingTitle).toBe('str5');
     expect(resp.lastJson.waitingSubtitle).toBe('str6');
     expect(resp.lastJson.logoUrl).toBe('str7');
+    expect(resp.lastJson).not.toHaveProperty('cosmosDBConnectionString');
+    expect(resp.lastJson).not.toHaveProperty('cosmosDBEndpoint');
+    expect(resp.lastJson).not.toHaveProperty('cosmosDBName');
```

Comment:
I think these assertions are more suited for the `getClientConfig` tests, since `getClientConfig` is being invoked by configController.